Sources/Commands/SwiftTool.swift

```diff
@@ -46,6 +48,10 @@ public class SwiftTool<Mode: Argument, OptionType: Options> {
         let dynamicSelf = type(of: self)
         do {
             (self.mode, self.options) = try dynamicSelf.parse(commandLineArguments: args)
+            // Honor chdir option is provided.
+            if let dir = options.chdir {
+                try chdir(dir.asString)
```

Comment:
I agree, this can become part of file system when `currentWorkingDirectory` moves there, I'll add a FIXME for now.